ACT.FFXIVTranslate/ACT.FFXIVTranslate/translate/microsoft/MicrosoftTranslateProvider.cs

```diff
@@ -148,18 +130,20 @@ private string GetAuthToken()
             return authToken;
         }
 
-        private void DoRequest(string authToken, string requestBody,
+        private void DoRequest(string authToken, string uri, string requestBody,
             out string responseBody, out HttpStatusCode statusCode)
         {
             string _responseBody;
             HttpStatusCode _statusCode;
 
+            ServicePointManager.SecurityProtocol |= SecurityProtocolType.Tls11 | SecurityProtocolType.Tls12;
```

Comment:
Is there a good reason to allow TLSv1.1 here?

Reply:
It seems the API endpoint does not support the default one. 
> "connection force closed by remote". 
Resolved after adding this line. Not sure if there is a better way.

Reply:
Interesting... I'd imagine it should support up to tlsv1.3. Anyway this looks fine to me.